.github/workflows/checks.yml

```diff
@@ -1,8 +1,8 @@
 name: "run-linting-checks"
 on:
-  pull_request:
```

Comment:
I suspect we'll continue developing this repo and adding pull requests, so maybe it's a good idea to keep the pull_request checks too? Will defer to the team.